docs/conf.py

```diff
@@ -18,8 +18,11 @@
     "python": ("https://docs.python.org/3", None),
     "numpy": ("https://numpy.org/doc/stable", None),
     "shapely": ("https://shapely.readthedocs.io/en/latest/", None),
+    "pyarrow": ("https://arrow.apache.org/docs/", None),
 }
 
+# TODO: seem to have no effect
+# (see https://github.com/sphinx-doc/sphinx/issues/10235)
```

Comment:
This comment is now no longer up to date? 
(the commit message in https://github.com/benbovy/spherely/pull/89/commits/c59675e82d612d9bcadd677540dd260a4c7ad9ee seemed to indicate that you changed to manual signatures so `autodoc_typehints = "none"` would actually work?)

Reply:
I think we can just remove `autodoc_typehints = "none"` now that we have switched to manual signatures globally.

The behavior is a bit confusing. IIUC when both `autodoc_docstring_signature=True` (default) *and* the 1st line of the docstrings looks like a signature, then Sphinx pops the 1st line and overrides the signature ignoring `autodoc_typehints` in the process (from https://github.com/sphinx-doc/sphinx/issues/10235#issuecomment-1059896282).